docs/sovereign-cloud/README.md

```diff
@@ -0,0 +1,67 @@
+# Sovereign Cloud Guide for Project Symphony
+
+This document highlights the steps necessary to test this project with a sovereign cloud.
```

Comment:
Perhaps we should add a line about switching between public clouds?

If you are using a registered cloud you can do so by using the -c option for rover which has built-in access to AzureCloud, AzureChinaCloud, AzureUSGovernment and AzureGermanCloud. Available here: [Rover](https://github.com/aztfmod/rover/blob/master/scripts/rover.sh)

Reply:
updated intro text with above suggested text, slightly revised.